### PR TITLE
Create Rule23-15.md

### PR DESCRIPTION
**Pull request by KarenWGard**

### CODE REVIEW

File: docs/section23/Rule23-15.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+1. P-RMR is modeled with at least one air-side system that includes humidistatic controls.  
+
+**Function Calls:**  
+
+
+**Applicability Checks:**  
+- For each hvac_p in the P_RMR: `for hvac_p in P_RMR...HeatingVentilatingAirConditioningSystem:`
+    - get the cooling system: `cooling_system_p = hvac_p.cooling_system`
+    - check to see if the cooling system has a humidification type other than NONE, if so, there are humidification controls and the rule is applicable: `if cooling_system_p.humidification_type != NONE:`
+        - the rule is applicable: `APPLICABLE`
```
Comment (mkarpman) [inline docs/section23/Rule23-15.md:25]: Should be "undetermined". Also see comment to 23-15

Comment (KarenWGard) [reply docs/section23/Rule23-15.md]: updated

File: docs/section23/Rule23-15.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    - if there are humidity controls, we will now check for zone humidity setpoint schedules: `if has_humidifcation_controls:`
+        - look through each zone served by the system: `for zone_id in hvac_systems_and_zones_p[hvac_p_id]["ZONE_LIST"]:`
+            - get the zone: `zone_p = get_object_by_id(zone_id)`
+            - if the zone_p.minimum_humidity_setpoint_schedule exists, the rule is applicable, return UNDETERMIEND: `if zone_p.minimum_humidity_setpoint_schedule: UNDETERMINED`
+            - if the zone_p.maximum_humidity_setpoint_schedule exists, the rule is applicable, return UNDETERMIEND: `if zone_p.maximum_humidity_setpoint_schedule: UNDETERMINED`
+    - if we make it down here without triggering an applicable result, the rule is not applicable: `NOT_APPLICABLE`
+
+
+**[Back](../_toc.md)**
+
```
[the remark this answers is not in the document]
Comment (KarenWGard) [reply docs/section23/Rule23-15.md:44]: updated

File: docs/section23/Rule23-15.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+1. get_dict_of_zones_and_terminal_units_served_by_hvac_sys
+
+
+**Applicability Checks:**  
+- get a list of the hvac systems and the zones they served in the P_RMR: `hvac_systems_and_zones_p = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(P_RMR)`
+- look through each hvac system in the proposed: `for hvac_p_id in hvac_systems_and_zones_p:`
+    - get the proposed hvac sytem: `hvac_p = get_object_by_id(hvac_p_id)`
+    - set a boolean has_humidifcation_controls equal to false: `has_humidifcation_controls = FALSE`
+    - get the heating system: `heating_system_p = hvac_p.heating_system`
+    - check to see if the heating system has a humidification type other than NONE, if so, there are humidification controls, set has_humidifcation_controls to TRUE: `if heating_system_p.humidification_type != NONE: has_humidifcation_controls = TRUE`
```
Comment (claperle) [inline docs/section23/Rule23-15.md:29]: I am not clear why we are checking for humification here and at the zone level if the rule applies to dehumification

**Review by claperle — APPROVED** [review]

**Pull request merged**